scripts/portstat

```diff
@@ -333,13 +333,13 @@ class Portstat(object):
                               format_number_with_comma(data['tx_err']),
                               format_number_with_comma(data['tx_drop']),
                               format_number_with_comma(data['tx_ovr'])))
-
-        if use_json:
-            print(table_as_json(table, header))
-        else:
-            print(tabulate(table, header, tablefmt='simple', stralign='right'))
-            if multi_asic.is_multi_asic() or device_info.is_chassis():
-                print("\nReminder: Please execute 'show interface counters -d all' to include internal links\n")
+        if table:
```

Comment:
if `table` is empty, can we still print the reminder?

Reply:
ok, yes make sense. let me update

Reply:
@wenyiz2021 updated

Reply:
thanks, can you please add the `if not use_json` as well on line 341

Reply:
line 341 should be `if multi_asic.is_multi_asic() or device_info.is_chassis() and not use_json:`
if `use_json` is true, the cli expect json format output and we won't want the reminder string

Reply:
@wenyiz2021 updated.